easybuild/easyconfigs/p/pypmt/pypmt-1.1.0-foss-2022a.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'CMakeMake'
+
+name = 'pypmt'
+version = '1.1.0'
+
+homepage = 'https://git.astron.nl/RD/pmt'
+description = """PMT is a high-level software library capable of 
+    collecting power consumption measurements on various hardware."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+builddependencies = [('CMake', '3.23.1')]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+]
+
+sources = [{
+    'filename': 'pmt-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://git.astron.nl/RD',
+        'repo_name': 'pmt',
+        'tag':  '%(version)s'
+    },
+}]
+
+checksums = ['d6332ce193dfdbc8073ba1fb600d1cebbd0a17c00f650003fab044aa721faf70']
```

Comment:
Checksum is failing for me, I think because with this source specification, it will do a git clone and create a tarball. Those checksums are not reproducable. However, I see that there is a direct download link as well: https://git.astron.nl/RD/pmt/-/archive/1.1.0/pmt-1.1.0.tar.gz
Can you use that? That _should_ give a reproducible checksum, which is preferable.